public/pages/Findings/components/FindingDetailsFlyout.tsx

```diff
@@ -140,51 +135,75 @@ export default class FindingDetailsFlyout extends Component<
             </EuiFormRow>
 
             <EuiSpacer size={'l'} />
+          </EuiAccordion>
+          {rules.length > 1 && <EuiHorizontalRule />}
+        </div>
+      );
+    });
+  };
+
+  onViewSurroundingDocumentsClicked = () => {};
```

Comment:
This will be implemented in next PR where the discover plugin will get integrated

Reply:
Added in this PR itself